client/wfnews-war/src/main/angular/src/app/components/report-of-fire/report-of-fire.config.json

```diff
@@ -48,6 +48,20 @@
       "updateAttribute": ["fullName", "phoneNumber"],
       "showProgress": false,
       "previousId": "callback-page",
+      "nextId": "compass-page"
+    },
+    {
+      "id": "compass-page",
+      "type": "RoFCompassPage",
+      "title": "What direction is the fire?",
+      "message": "Hold your phone steady and orient it so that the yellow arrow on the compass is pointing in the direction of the fire.",
+      "allowSkip": false,
```

Comment:
according to Figma, this step is skippable.

Reply:
Thanks I updated that now